analysis_driver/notification.py

```diff
@@ -1,4 +1,59 @@
-from egcg_core import notifications
+from egcg_core import notifications, clarity
+from pyclarity_lims.lims import Step
```

Comment:
This should be pyclarity_lims.entities. Also, this should be added to requirements.txt